src/main/resources/assets/neoforge/lang/en_us.json

```diff
@@ -263,5 +263,21 @@
 
   "neoforge.network.extensible_enums.no_vanilla_server": "This client does not support vanilla servers as it has extended enums used in serverbound networking",
   "neoforge.network.extensible_enums.enum_set_mismatch": "The set of extensible enums on the client and server do not match. Make sure you are using the same NeoForge version as the server",
-  "neoforge.network.extensible_enums.enum_entry_mismatch": "The set of values added to extensible enums on the client and server do not match. Make sure you are using the same mod and NeoForge versions as the server. See the log for more details"
+  "neoforge.network.extensible_enums.enum_entry_mismatch": "The set of values added to extensible enums on the client and server do not match. Make sure you are using the same mod and NeoForge versions as the server. See the log for more details",
+
+  "neoforge.adv.base": "[Entity: %s | Item: %s]",
+  "neoforge.adv.sharpness_bonus": "[Sharp: +%s]",
```

Comment:
These should probably have `attribute` or something in their translation key. Maybe some of the other ones as well, though I can see the argument that some are rather generic so may not be worth restricting

Reply:
The attribute advanced debug info I can add `attribute` to.  The others don't really need it, they're pretty generic format strings.